students/Yue_Ma/Lesson03/mailroom.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+donor_list = {'Yue Ma': [100000, 1561132],
+              'Yanan Ma': [1000, 5645, 6161, 27],
+              'Jianqiang Ma': [200000, 854821, 1202],
+              'Chunhong Liu': [100.51, 1000.1],
+              'Robert Rowe': [20000000]}
+
+
+def main():
+    main_prompt = "\n".join(("Welcom to Mailroom!",
+                             "Please choose from below options:",
+                             "Enter: '1' - Send a Thank You note",
+                             "Enter: '2' - Create a report",
+                             "Enter: '3' - Quit the Mailroom"))
+    while True:
+        user_main = input(main_prompt)
+        if user_main == '1':
+            send_a_thank_you_note()
+            break
```

Comment:
so you want to remove your `break` statements because the idea is this is your main loop and will continue getting options and run your work functions.

Reply:
you'll want to remove `break` statement on lines 20 and 23 - this will allow your program to continue looping in while True loop.

You can keep break statement on line 26 to exit the program or you can map to a function that calls sys.exit() like we recommend in the guidelines.